src/main/java/gojava31.ua.goit/Category.java

```diff
@@ -0,0 +1,36 @@
+package gojava31.ua.goit;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+
+public final class Category {
+    private String name;
+    private List<Project> projects;
+
+    public Category() {
+    }
+
+    public Category(String name) {
+        this.name = name;
+    }
+
+    public Category(String name, List<Project> projects) {
+        this.name = name;
+        this.projects = projects;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public List<Project> getProjects() {
+        List<Project> clone = new CopyOnWriteArrayList<Project>();
+        Collections.copy(clone, projects);
```

Comment:
you can just pass the projects as a constructor arg to `CopyOnWriteArrayList`, all (or most) collections accept first arg being collection to copy from